app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -41,9 +41,30 @@ def setupv2
     not_found unless group_import_resiliency_enabled?
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable MethodLength
   def create_repo
-    raise NotImplementedError
+    will_create_repo =
+      if group_invite_status.accepted? || group_invite_status.errored?
+        if group_assignment_repo&.github_repository&.empty?
+          group_assignment_repo&.destroy
+          @group_assignment_repo = nil
+        end
+        report_retry
+        group_invite_status.waiting!
+        GroupAssignmentRepo::CreateGitHubRepositoryJob.perform_later(group_assignment, group, retries: 3)
+        true
+      else
+        false
+      end
+    render json: {
+      will_create_repo: will_create_repo,
```

Comment:
Is the field here different from the `job_started` field in `create_repo` in `AssignmentRepoInvitationsController`? Might be easier to follow in the future if they both use the same field name?

Reply:
TL;DR : I will change it back to `job_started`

The field is the exact same idea. At the time of writing this I thought `will_create_repo` sounded more descriptive and less confusing but now I feel that `job_started` is better because it's context is from within the `PUT /create_repo` route. I will change it back to `job_started`.